### ISSUE TITLE
Feature request: transaction types

### ISSUE THREAD

**Issue by PaulRBerg** [smlxl/evm.codes#270]
It would be helpful to have a section on the main page (or a new page) about transaction types. IIRC there are three types at the moment:

- Type 0: Legacy
- Type 1: EIP-2718 with access lists
- Type 2: EIP-1559 (also with access lists)

This is an important feature request given the rise of rollups, which often impoemented new tx types (e.g. [Arbitrum](https://www.rollup.codes/arbitrum-one))

evm.codes could be a baseline against which to compare these new rollup tx types.

**Comment by dorlevi** [comment]
I agree that this would be super useful, I'm not sure we have a lot of capacity atm as we are working on evm.storage but if someone sees it and wants to take it on, we will compensate with a bounty.

**Comment by developerfred** [comment]
Thanks @dorlevi ! 

Hi @PaulRBerg, I'm working on this issue and I'd like to know if you have any more suggestions for it since it's been a few months since you opened the issue.

**Comment by PaulRBerg** [comment]
Don't have any other concrete suggestions, unfortunately

**Comment by dorlevi** [comment]
I think this one is pretty straight forward, explanation of the different type, maybe evm.storage links to different transactions of diff types  and then like different edgecases that could happen in each tx types

cc: @thevaizman @peroket  if they have ideas

**Comment by peroket** [comment]
I would also add why this type of transaction has been created, its purpose.

**Comment by developerfred** [comment]
![Uploading Captura de Tela 2024-01-04 às 20.35.42.png…]()

**Comment by developerfred** [comment]
<img width="1278" alt="Captura de Tela 2024-01-04 às 20 35 42" src="https://github.com/smlxl/evm.codes/assets/57037080/2f9f3a00-d3c4-453e-af6e-f8df8baab046">

**Comment by developerfred** [comment]
![Uploading Captura de Tela 2024-01-04 às 20.58.46.png…]()

**Comment by developerfred** [comment]
<img width="1270" alt="Captura de Tela 2024-01-04 às 20 58 46" src="https://github.com/smlxl/evm.codes/assets/57037080/f5b45dd6-bceb-41e7-9890-64706f4c6524">
